document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -40,69 +36,42 @@
 
     private static final String SMALL_FULL = "smallfull";
 
+    private static final String ABRIDGED_REGEX = "/transactions\\/[0-9-]+/accounts\\/.*";
+
+    private static final String SMALL_FULL_REGEX = "/transactions\\/[0-9-]+/company-accounts\\/.*";
```

Comment:
This link is for company accounts, not specifically small-full. Please can you change to:
```java
private static final String ACCOUNTS_REGEX = "/transactions\\/[0-9-]+/accounts\\/.*";
private static final String COMPANY_ACCOUNTS_REGEX = "/transactions\\/[0-9-]+/company-accounts\\/.*";
```